indico/htdocs/sass/modules/_event_display.scss

```diff
@@ -100,32 +100,76 @@ li > table + .note-area-wrapper {
         font-size: 1.1em;
     }
 }
+.participant-list-wrapper{
+    position: relative;
+
+    &.collapsable{
+        &.collapsed {
+            max-height: 4.75em;
+            overflow: hidden;
+
+            .gradient-layer {
+                @include gradient(rgba(0,0,0,0.0), #ECECEC);
```

Comment:
To save you the trouble: `#ECECEC` doesn't exist in the palette and it's used in the meeting view.